src/control/jobs/CustomExportJob.cpp

```diff
@@ -268,15 +280,19 @@ void CustomExportJob::run()
 		// don't lock the page here for the whole flow, else we get a dead lock...
 		// the ui is blocked, so there should be no changes...
 		Document* doc = control->getDocument();
-
+		
 		XojPdfExport* pdfe = XojPdfExportFactory::createExport(doc, control);
-
+		
+		if (this->choosenFilterName == EXPORT_PDF_NOBG)
+		{
+			pdfe->setNoBackgroundExport(true);
+		}
 
 #ifdef ADVANCED_PDF_EXPORT_POPPLER
 		// Not working with ADVANCED_PDF_EXPORT_POPPLER
 		if (!pdfe->createPdf(this->filename))
 #else
-		if (!pdfe->createPdf(this->filename, exportRange))
+		if (!pdfe->createPdf(this->filename/*, exportRange*/))	//export range is called anyways
```

Comment:
With this you probably break the Export with only some Page ranges (not PDF Export, export and Select PDF), did you test that?

Reply:
Ok then there's something wrong... if I leave there `exportRange` an try to export, the output pdf is empty

Reply:
Oh, no, ok.. there must be an index bound problem because the first page is always absent when exporting all pages

Reply:
The first page has the Index 0
But the user enters page 1, because a normal user starts counting at 1, not 0. Only computer Scientists do :-)